app/views/admin/users/show.html.erb

```diff
@@ -1,50 +1,37 @@
-<div class="grid gap-6">
-  <div class="crayons-card p-6">
-    <h2 class="d-inline">
-      <% if @user.registered? %>
-        <%= @user.name %><%= link_to "@#{@user.username}", @user.path, class: "ml-2", target: "_blank", rel: "noopener" %>
-      <% else %>
-        @<%= @user.username %> has not accepted their invitation yet.
-      <% end %>
-    </h2>
-    <% if @user.registered? %>
-      <a href="<%= edit_admin_user_path(@user) %>" class="btn btn-primary float-right">Manage User</a>
-      <p class="font-italic">Member since <%= @user.created_at.strftime("%b %e '%y") %></p>
-    <% end %>
+<% if true %>
+  <%= render "header" %>
+  <%= render "navigation" %>
 
-    <hr>
+  <div class="crayons-card p-7"><%# This container could be considered as wrapper for tab's content %>
+    <%= render "activity_new" if @user.registered %>
 
-    <h2>General Info</h2>
-    <dl>
-      <dt>ID:</dt>
-      <dd><%= @user.id %></dd>
-      <dt>Username:</dt>
-      <dd><%= @user.username %></dd>
-      <dt>Name:</dt>
-      <dd><%= @user.name %></dd>
-      <dt>Email:</dt>
-      <dd><%= @user.email %></dd>
-      <dt>Access Locked:</dt>
-      <dd><%= @user.access_locked? %></dd>
-      <dt>Twitter:</dt>
-      <% if @user.twitter_username %>
-        <dd><%= link_to @user.twitter_username, "https://twitter.com/#{@user.twitter_username}" %></dd>
-      <% else %>
-        <dd>N/A</dd>
-      <% end %>
-      <dt>GitHub:</dt>
-      <% if @user.github_username %>
-        <dd><%= link_to @user.github_username, "https://github.com/#{@user.github_username}" %></dd>
-      <% else %>
-        <dd>N/A</dd>
-      <% end %>
-    </dl>
-  </div>
+    <div class="m:grid gap-8 grid-cols-2">
+      <div>
+        <div class="mb-7">
```

Comment:
I decided to keep the overall structure of widgets OUTSIDE the partial so it's easier to see and update general layout - you'd do it in one file instead of each widget's file separately.